sdk/identity/identity/README.md

```diff
@@ -64,18 +68,35 @@ The `DefaultAzureCredential` is appropriate for most scenarios where the applica
 - Visual Studio Code - If the developer has authenticated via the Visual Studio Code Azure Account plugin, the `DefaultAzureCredential` will authenticate with that account.
 - Azure CLI - If the developer has authenticated an account via the Azure CLI `az login` command, the `DefaultAzureCredential` will authenticate with that account.
 
-### Environment variables
+## Environment Variables
+
+[`DefaultAzureCredential`][ref_defaultazurecredential] and [`EnvironmentCredential`][ref_environmentcredential] can be configured with environment variables. Each type of authentication requires values for specific variables:
```

Comment:
Do ref_defaultazurecredential and ref_environmentcredential exist? I couldn't find them elsewhere in this file 🤔

Reply:
Ooops good catch. I didn't copy/paste those